corehq/apps/app_manager/static/app_manager/js/section_changer.js

```diff
@@ -34,8 +34,10 @@
  */
 hqDefine("app_manager/js/section_changer", [
     'jquery',
+    'underscore',
 ], function (
-    $
+    $,
+    _,
```

Comment:
Does the comma here cause issues? Aka is there a reason we didn't have it there before?

Reply:
ES5 doesn't allow trailing commas for parameters lists in function calls, that was added in ES6. It's not a problem in browsers, which have moved on from ES5, but for code using requirejs, it'll break deploy (because uglify will fail to parse the file). HQ has been using ES5 in linting rules forever for this reason. 

https://github.com/dimagi/commcare-hq/pull/35651 updated HQ to lint using ES6, now that almost all of our code is off of requirejs. The new lint config expects trailing commas in function calls.

So it should have been there before, I just haven't gotten in the habit yet...that might also be more evidence that I did this file by hand before scripting, since spot checking the related PRs, they have trailing commas for the `hqDefine` callback arguments.